sklearn/covariance/shrunk_covariance_.py

```diff
@@ -365,6 +368,23 @@ class LedoitWolf(EmpiricalCovariance):
         Coefficient in the convex combination used for the computation
         of the shrunk estimate.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.covariance import LedoitWolf
+    >>> real_cov = np.array([[.4, .2],
+    ...                      [.2, .8]])
+    >>> np.random.seed(0)
+    >>> X = np.random.multivariate_normal(mean=[0, 0],
+    ...                                   cov=real_cov,
+    ...                                   size=50)
+    >>> cov = LedoitWolf().fit(X)
+    >>> cov.covariance_
+    array([[0.4406..., 0.1616...],
+           [0.1616..., 0.8022...]])
+    >>> cov.location_
+    array([0.0595... , -0.0075...])
```

Comment:
I think you're missing a space before `0.0595...`, that's why travis is failing. Could you also add ` # doctest: +ELLIPSIS` at the end of the `cov.covariance_` line? You can read about it [here](https://docs.python.org/3/library/doctest.html#directives)